index.markdown

```diff
@@ -296,6 +296,7 @@ redirect_from: '/moveit/'
                 <!--         </div> -->
                 <!--     </div> -->
                 <!-- </div> -->
+		<!--
```

Comment:
Why is this being removed? It should be updated to MoveIt H, which also requires changing the dates in the javascript. Please undo

Reply:
Also, we should use this same HTML on the roadmap and homepage, by using an _include_ file.